CompanyApi/apis.http

```diff
@@ -58,4 +58,74 @@ Content-Type: application/json; charset=utf-8
 ### Case1: Sucess, 204 NO Content
 ### Case2: 404 Company NOT Found
 
+######################################################################################
+
+######################################################################################
+
+## AC6 add employee 
+POST  https://{{hostname}}:{{port}}/api/Companies/{companyId}

```

Comment:
the api can be https://{{hostname}}:{{port}}/api/Companies/{companyId}/employees